TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utils/RobotConfig.kt

```diff
@@ -0,0 +1,39 @@
+@file:Suppress("ktlint:standard:property-naming")
+
+package org.firstinspires.ftc.teamcode.utils
+
+import org.firstinspires.ftc.teamcode.utils.RobotConfig.debug
+
+/**
+ * This is an immutable object representing robot configuration.
+ *
+ * It is meant to orchestrate FTC Dashboard and other configuration together.
+ * Certain sub-objects are annotated with `@Config`. This designates them as FTC Dashboard
+ * configuration that can be modified at runtime. **To permanently change these values, you must
+ * also modify the code!** The configuration can also change during initialization depending on
+ * various build constants like [debug].
+ */
+object RobotConfig {
+
+    /**
+     * When true, enables debugging features like camera streaming and more logs.
+     *
+     * This should be disabled for competitions.
+     */
+     const val debug: Boolean = true
```

Comment:
Should this be made SCREAMING_SNAKE_CASE too? It's a constant like the other config values. I'm not sure why this wasn't capitalized to begin with.